cpp/memilio/math/README.md

```diff
@@ -7,7 +7,7 @@ This module contains different mathematical methods for integration of sets of o
 The model consists of the following classes:
 1. Integrator: The integrator module contains an IntegratorCore and an OdeIntegrator class, currently designed for explicit methods only. The IntegratorCore contains a step function that takes as input the right hand side function of the ODE, the current time, the step size etc. It represents a generic integration method from which the other integrators are derived from. The OdeIntegrator stores an IntegratorCore as well as the right hand side function and a time series of results.
 2. The following integrators implement IntegratorCore and can be used with OdeIntegratorCore or [mio::Simulation](../compartments/README.md).
-    - ControlledStepperWrapper: The ControlledStepperWrapper class allows using integrators from boosts::numeric::odeint in memilio. The integrator is passed as template argument. It must be of the concept "Error Stepper" (for more details, see [boost's own documentation](https://www.boost.org/doc/libs/1_75_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/odeint_in_detail/steppers.html#boost_numeric_odeint.odeint_in_detail.steppers.stepper_overview) ). Currently, the default for mio::Simulation is runge_kutta_cash_karp54. Alternatively, runge_kutta_dopri5 or runge_kutta_fehlberg78 can be used.
+    - ControlledStepperWrapper: The ControlledStepperWrapper class allows using integrators from boosts::numeric::odeint in memilio. The integrator is passed as template argument. It must be of the concept "Error Stepper" (for more details, see [boost's own documentation](https://www.boost.org/doc/libs/1_84_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/odeint_in_detail/steppers.html#boost_numeric_odeint.odeint_in_detail.steppers.stepper_overview) ). Currently, the default for mio::Simulation is runge_kutta_cash_karp54. Alternatively, runge_kutta4 or runge_kutta_fehlberg78 can be used.
```

Comment:
Is this true? Can we actually use the fixed dt stepper `runge_kutta4 `?

Reply:
Oh, I thought it was adaptive as well. In that case, no, it doesn't work.
```suggestion
    - ControlledStepperWrapper: The ControlledStepperWrapper class allows using integrators from boosts::numeric::odeint in memilio. The integrator is passed as template argument. It must be of the concept "Error Stepper" (for more details, see [boost's own documentation](https://www.boost.org/doc/libs/1_84_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/odeint_in_detail/steppers.html#boost_numeric_odeint.odeint_in_detail.steppers.stepper_overview) ). Currently, the default for mio::Simulation is runge_kutta_cash_karp54. Alternatively, runge_kutta_fehlberg78 can be used.
```